inst/RcppR6_classes.yml

```diff
@@ -749,21 +745,22 @@ StochasticPatchRunner:
       access: function
       name_cpp: "plant::get_state"
 
-Canopy:
-  name_cpp: "plant::Canopy"
+Resource_spline:
+  name_cpp: "plant::Resource_spline"
   constructor:
     args:
       - tol: double
       - nbase: double
       - max_depth: double
+      - rescale_usually: bool
   methods:
-    canopy_openness:
+    get_value_at_height:
```

Comment:
Possibly, "get_resource_value_at_height"?

Reply:
Possible, but I'll leave as is. The specification of a resource actually comes when you call it, e.g. 

`ight_availability.get_value_at_height`